app/controllers/api/v1/targets_controller.rb

```diff
@@ -1,6 +1,10 @@
 module Api
   module V1
     class TargetsController < Api::V1::ApiController
+      def new
+        @topics = Topic.all
+      end
```

Comment:
The new action should be reserved to create display a form for the creation of a new resource. At this stage of the project, you could either create a `Topics` controller with the index action, or use the Rails console for development/testing purposes only.

Reply:
In this case, we decide to do it in that way, because topics are used just to create a target, we think that it was not necessary to create a new controller just for index action.